src/CMakeLists.txt

```diff
@@ -77,6 +77,23 @@ if (NOT SPADES_GIT_SHA1)
   set(SPADES_GIT_SHA1 "N/A")
 endif()
 
+file(STRINGS "${CMAKE_SOURCE_DIR}/../VERSION" VERSION_STRING LIMIT_COUNT 1)
+string(REGEX MATCH "([a-zA-Z]*)-\?([0-9]+)\.([0-9]+)\.([0-9]+)(.*)" ver "${VERSION_STRING}")
+if (ver STREQUAL "")
+  message(FATAL_ERROR "Error to parse version string from ${VERSION_STRING}")
+endif()
+if (CMAKE_MATCH_1 STREQUAL "")
+  set(SPADES_FLAVOUR "SPAdes")
+else()
+  set(SPADES_FLAVOUR "${CMAKE_MATCH_1}")
```

Comment:
CMAKE_MATCH_N looks like a creepy logic to someone who is not that familiar with cmake :)

Reply:
Yeah. Very cmake-ish.